src/test/ui-fulldeps/session-diagnostic/diagnostic-derive.rs

```diff
@@ -740,14 +732,6 @@ pub(crate) struct SubdiagnosticWithSuggestion {
     correct: String,
 }
 
-#[derive(Diagnostic)]
-#[diag(compiletest_example)]
-struct SubdiagnosticEagerSuggestion {
-    #[subdiagnostic(eager)]
-    //~^ ERROR `#[subdiagnostic(...)]` is not a valid attribute
-    sub: SubdiagnosticWithSuggestion,
-}
-
```

Comment:
You can just leave these structs but remove the error annotations to test this.

Reply:
Done :)